Marlin/Marlin_main.cpp

```diff
@@ -1019,6 +1035,22 @@ inline void get_serial_commands() {
 
       // If command was e-stop process now
       if (strcmp(command, "M112") == 0) kill(PSTR(MSG_KILLED));
+      #if ENABLED(PROBE_FAIL_PANIC)
+        if (strcmp(command, "G26") == 0) {
+          LCD_MESSAGEPGM(WELCOME_MSG);
+          probe_fail = false;
+        }
+        else if(probe_fail && (strchr(command, 'G') != NULL || strchr(command, 'M') != NULL) && strstr(command, "M105") == NULL) {
+          for(uint8_t j=0; j < MAX_CMD_SIZE; j++)
+            command_queue[cmd_queue_index_r][j] = 0;
+          if(commands_in_queue) {
+            commands_in_queue = 1;
+            cmd_queue_index_r = (cmd_queue_index_r + 1) % BUFSIZE;
+          }
+          serial_count = 0;
+          return;
+        }
+      #endif
```

Comment:
@jbrazio @thinkyhead please take a look at this snippet. In case the probing fails all attempts `G26` is used to give control back to the host, all other commands are ignored so that the print doesn't process. This is managed in the parser and is intended to handle hosts that aren't aware of this feature.

We prefer to reserve `G26` if possible.

Granted if the intent is to be able to do manual cleaning of the nozzle and continue then the host needs to prevent all G-Code from being sent other than `G26`, Marlin cannot hold each command it receives in its buffer until it sees a `G26`, not enough memory; and it shouldn't throw away all other commands if they're not `G26` as done here. This only done to handle "unaware" hosts.